resources/js/Components/MismatchRow.vue

```diff
@@ -3,12 +3,13 @@
         <td :data-header="$i18n('column-property')">
             <wikit-link
               class="break-line-link"
+              target="blank"
```

Comment:
```suggestion
              target="_blank"
```

Reply:
oh, yes, i had the feeling i had that made that mistake.